user_form.html

```diff
@@ -104,3 +104,11 @@
 {% if templateset.custom_fields.site_addl_js_url %}
 	<script type="text/javascript" src="{{ templateset.custom_fields.site_addl_js_url }}"></script>
 {% endif %}
+
+<script>

```

Comment:
@sukhada could you add code comments about what's happening here? Is this dynamically reordering the forms fields? Also, shouldn't these be a class target and not an ID, as _box is in the classnames here?

Reply:
@cjmabry I removed this; this is no longer necessary because of the other changes that were made